tests_e2e/orchestrator/scripts/prepare-pypy

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
So, this is a standalone tool, someone needs to run to update the image?

Reply:
it's described in the wiki